### DISCUSSION TITLE
Improved integration of dark theme and light theme with gtk applications

### DISCUSSION THREAD

**Discussion by JoaoGVTheodoro** [elementary/triage#351] [Ideas]
Currently in Elementary OS, the system's custom colors are not integrated into many GTK3 or 4 applications, as this detail in the KDE interface is notorious. The colors in Pantheon are still very limited, it would be great if this was corrected and all apps supported the system layout so as not to see a break in the interface design!

**Comment by danirabbit** [comment]
Pantheon uses both the freedesktop dark style preference and the freedesktop accent color preference already. So if there's an app that isn't picking that up correctly, it's something you should report to the app developer :)

**Reply by JoaoGVTheodoro** [reply]
ok, thank you very much for your attention! I'll do it! Success :)
